src/TurbulenceClosures/turbulence_closure_implementations/leith_enstrophy_diffusivity.jl

```diff
@@ -86,9 +86,9 @@ const ArrayOrField = Union{AbstractArray, AbstractField}
     return wxz² + wyz²
 end
 
-@inline calc_νᶜᶜᶜ(i, j, k, grid, closure::TwoDimensionalLeith{FT}, buoyancy, U, C) where FT =
-    (closure.C * Δᶠ(i, j, k, grid, closure))^3 * sqrt(  abs²_∇h_ζ(i, j, k, grid, U)
-                                              + abs²_∇h_wz(i, j, k, grid, U.w))
+@inline calc_nonlinear_νᶜᶜᶜ(i, j, k, grid, closure::TwoDimensionalLeith{FT}, buoyancy, velocities, tracers) where FT =
+    (closure.C * Δᶠ(i, j, k, grid, closure))^3 * sqrt(  abs²_∇h_ζ(i, j, k, grid, velocities)
+                                              + abs²_∇h_wz(i, j, k, grid, velocities.w))
```

Comment:
Some alignment has been broken here. I think it's supposed to be

```suggestion
@inline calc_nonlinear_νᶜᶜᶜ(i, j, k, grid, closure::TwoDimensionalLeith{FT}, buoyancy, velocities, tracers) where FT =
    (closure.C * Δᶠ(i, j, k, grid, closure))^3 * sqrt(   abs²_∇h_ζ(i, j, k, grid, velocities)
                                                      + abs²_∇h_wz(i, j, k, grid, velocities.w))
```